### DISCUSSION TITLE
Potential Security Vulnerability disclosure

### DISCUSSION THREAD

**Discussion by dmarlingSplunk** [danny-avila/LibreChat#1745] [Feature Requests & Suggestions]
### What happened?

Hello, I would like to make a responsible disclosure of some vulnerabilities that I have found.  Please let me know the best method you would like me to disclose this to you. Be aware that I cannot use discord unfortunately. Thank you! 

### Steps to Reproduce

N/A

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Thanks! Feel free to email me which you can find on my profile page. Im curious what it may be since I recently did a thorough security review.

**Comment by dmarlingSplunk** [comment]
Sending the email now.  Thank you for the quick response.

**Reply by danny-avila** [reply]
Of course. Security is a top concern. Thanks for reporting/looking into this.
